spec/services/notifications/moderation/send_spec.rb

```diff
@@ -10,7 +10,7 @@
   let(:moderator) { available_moderators.first }
 
   before do
-    u = create(:user, :trusted, last_moderation_notification: last_moderation_time)
+    u = create(:user, :trusted, last_reacted_at: 2.days.ago, last_moderation_notification: last_moderation_time)
```

Comment:
I could add this to the user factory, but I didn't because I think that adding this attribute in the spec leads us to the app/services/notifications/moderation.rb